main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -173,7 +177,23 @@ DataTableView.prototype._renderPagingControls = function(pageSizeControls, pagin
     previousPage.addClass("inaction");
   }
 
-  $('<span>').addClass("viewpanel-pagingcount").html(" " + from + " - " + to + " ").appendTo(pagingControls);
+  lastPageSpan = $('<span>').attr("id", "viewpanel-paging-last").text(self._lastPageNumber);
+  pageControlSpan = $('<span>').attr("id", "viewpanel-paging-current");
+  
+  currentPageInput = $('<input type="number">')
+    .change(function(evt) { self._onChangeGotoPage(this, evt); })
+    .attr("id", "viewpanel-paging-current-input")
+    .attr("min", 1)
+    .attr("max", self._lastPageNumber)
+    .attr("id", "viewpanel-paging-current-input")
```

Comment:
I'd also add `required` so we don't accept an empty value
```suggestion
    .attr("required") 
```
REF: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#Validation